### PR TITLE
Venafi demo

### LINKED ISSUE #23: Create a dockerised demo

### PR DESCRIPTION
**Pull request by jbowkett**
Fixes #23 and #22

### CODE REVIEW

File: config/source-quickstart.properties [hunk 1]
```diff
@@ -5,9 +5,9 @@ tasks.max=1
 # The remaining configs are specific to the Venafi source connector. In this example, we connect to
 # the VENAFI VEDSDK API on localhost using username "admin" and password "admin". These values should be changed for
 # your specific instance.
-venafi.base.url=https://localhost:443/VEDSDK
-venafi.username=admin
-venafi.password=admin
+venafi.base.url=https://uvo1ys1ukgf7v8x02y2.env.cloudshare.com
+venafi.username=tppadmin
+venafi.password=Password123!
```
Comment (afaedda) [inline config/source-quickstart.properties:11]: This probably should go back to how it was

File: README.md [hunk 2]
```diff
@@ -61,17 +61,67 @@ If you intend to change the JAR please stop, change the JAR, then start the clus
...
+2. Kafka server:
+kafka-connect-venafi/kafka-install/kafka_2.13-2.4.0/bin % ./kafka-server-start.sh ../config/server.properties
+3. Kafka connect:
+kafka-connect-venafi % kafka-install/kafka_2.13-2.4.0/bin/connect-standalone.sh kafka-install/kafka_2.13-2.4.0/config/connect-standalone.properties config/source-quickstart.properties
+
+^^ this property file is attached, and is an edited version of the main github source tree for the connector property file with the credentials for the cloudshare VM given to me by Venafi
+
+4. Kafka console consumer (to see the messages):
+kafka-connect-venafi/kafka-install/kafka_2.13-2.4.0/bin % ./kafka-console-consumer.sh --bootstrap-server localhost:9092 --topic VENAFI-LOGS
+
```
Comment (jbowkett) [inline README.md:99]: Can you remove some of these bogus lines in the readme - they're just notes of mine.  Maybe update the readme to just show someone how to use the docker image?

File: changelog.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+
```
Comment (jbowkett) [inline changelog.md:1]: Is this the version you have ready?....You should add in the compatiblity with the Venafi version.  Maybe tie our version of the connector to the Venafi version?...i.e. 21.3 perhaps?

File: demo/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,21 @@
+FROM openjdk:11
```
Comment (jbowkett) [inline demo/Dockerfile:1]: Is this your dockerfile, or one I was mucking about with?....if it's mine, then probably delete it, as yours is almost certainly much better?

File: demo/connect-standalone.properties [hunk 5]
```diff
@@ -0,0 +1,41 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
```
Comment (jbowkett) [inline demo/connect-standalone.properties:1]: is this yours or mine?...if mine, delete it if it's not needed

File: demo/startup-script.sh [hunk 6]
```diff
@@ -0,0 +1,46 @@
+#! /bin/bash
```
Comment (jbowkett) [inline demo/startup-script.sh:1]: This looks like something I did and can probably be deleted

File: docker-compose-2.yml [hunk 7]
```diff
@@ -0,0 +1,934 @@
+# docker-compose supports environment variable substitution with the ${VARIABLE-NAME} syntax.
```
Comment (jbowkett) [inline docker-compose-2.yml:1]: This looks liek something that can be deleted

File: docker-compose-confluent.yml [hunk 8]
```diff
@@ -0,0 +1,933 @@
+# docker-compose supports environment variable substitution with the ${VARIABLE-NAME} syntax.
```
Comment (jbowkett) [inline docker-compose-confluent.yml:1]: delete?

File: docker-compose.yml [hunk 9]
```diff
@@ -0,0 +1,246 @@
+version: '3.8'
```
Comment (jbowkett) [inline docker-compose.yml:1]: This looks like something you have added?....If so keep it, if not then bin it if it's not used

File: integration/Dockerfile [hunk 10]
```diff
@@ -0,0 +1,16 @@
+# we start from a Kafka image as the connector is in Kafka distribution
```
Comment (jbowkett) [inline integration/Dockerfile:1]: This looks like the one true dockerfile we need - if it works as a demo, then we should bin the rest please

**Review by jbowkett — COMMENTED** [review]
Smal changes requested, mainly just tidying up my mess :(

File: README.md (hunk 2 above)
Comment (afaedda) [reply README.md]: done

File: changelog.md (hunk 3 above)
Comment (afaedda) [reply changelog.md:1]: yes, 0.9.6 is the version that's going to master, but I don't get what you mean here!

File: demo/Dockerfile (hunk 4 above)
Comment (afaedda) [reply demo/Dockerfile]: removed all docker-compose/Dockerfile not used

File: changelog.md (hunk 3 above)
Comment (afaedda) [reply changelog.md:1]: Where did you get 21.3? I can see the Swagger in the VM has version 20.4...

Comment (jbowkett) [reply changelog.md:1]: Oh sorry, I thought their release numbers followed the current calendar year....go for 20.4 then?

**Review by afaedda — DISMISSED** [review]

**Review by rufusfnash — APPROVED** [review]

**Pull request merged**